Gemfile.lock

```diff
@@ -1,13 +1,21 @@
 PATH
   remote: .
   specs:
-    fake-kafka (0.0.2)
```

Comment:
That's an option, bundler also recommends to leave it in some cases as explained [here](https://bundler.io/guides/faq.html#using-gemfiles-inside-gems). Even though it's true that at Catawiki we usually remove it, I decided to keep it here since it was originally included.

I'm not against going for it 😄  But I think it'd be better if it's done in a separate PR.

Reply:
Maybe remove it then in a pre-work PR (which we can approve quickly)? That would simplify this PR's diff, right?

Reply:
Sure, if there is already a general agreement on removing the `Gemfile.lock` among the new maintainers, I'll be happy to do it 😄

Reply:
Done: https://github.com/catawiki/fake-kafka/pull/22